debian/changelog

```diff
@@ -1,12 +1,31 @@
-cloud-init (22.4-0ubuntu0~18.04.2) UNRELEASED; urgency=medium
+cloud-init (23.1-0ubuntu0~18.04.2) bionic; urgency=medium
+
+  * d/patches/netplan99-cannot-use-default.patch:
```

Comment:
+1 on this changeset and the quilt patch to disable use of 'to: default'  and bumping cloud-init version to 23.1-0ubuntu0~18.04.2